inc/tag.class.php

```diff
@@ -701,14 +702,17 @@ public static function showTagDropdown($params = [])
             $token_creation = "return null;";
         }
 
-        $readOnly = !$tag::canUpdate()
-            || ($obj->isNewItem() && !$obj->canCreateItem())
-            || (!$obj->isNewItem() && !$obj->canUpdateItem())
-        ;
+        $all_itemtype_can_update = count(array_filter($params['items_ids'], function ($value) use ($obj) {
+            $obj->getFromDB($value);
+            return !$obj->canUpdateItem();
+        })) === 0;
 
-        if (is_array($params['id']) && $tag::canUpdate()) {
-            $readOnly = false;
-        }
+        $readOnly = !$tag::canUpdate()
+            || (empty($params['items_ids']) && (
+                ($obj->isNewItem() && !$obj->canCreateItem())
+                || (!$obj->isNewItem() && !$obj->canUpdateItem())
+            ))
+            || (isset($params['items_ids']) && !$all_itemtype_can_update);
```

Comment:
`isset($params['items_ids'])` is always true, because `items_ids` is set by default:
```suggestion
            || (!empty($params['items_ids']) && !$all_itemtype_can_update);
```